lib/modules/datasource/conda/readme.md

```diff
@@ -1,5 +1,34 @@
-This datasource returns releases from the specified [conda](https://docs.conda.io/en/latest/) registry.
+This datasource returns releases for package from anaconda registry. Third-part repo is not supported currently.
```

Comment:
```suggestion
This datasource returns releases for package from anaconda registry. Third-party repo is not supported currently.
```

Reply:
Emmm this is not very correct, we do support prefix.dev which I think should be included as third party reoo

Reply:
I forget to update this file